openedx/core/djangoapps/user_api/accounts/api.py

```diff
@@ -42,6 +42,9 @@
 from openedx.core.djangoapps.user_api.preferences.api import update_user_preferences
 from openedx.core.lib.api.view_utils import add_serializer_errors
 
+# Import custom form model from campusromero_openedx_extensions plugin app.
```

Comment:
I think that we should use in some way the configuration of REGISTRATION_EXTENSION_FORM instead of this explicit import

Reply:
Yes, we could do that.

But these fields are not being used as part of the registration form anymore, they are only shown as part of the user's profile. Right now these fields are hidden using the front end but they are still present in the form.

 We should fix that right? We should remove them completely from the form and leave them only in the user's account @morenol